clients/drcachesim/docs/drcachesim.dox.in

```diff
@@ -1103,12 +1109,16 @@ Also, we preserved the following markers, but only for SYS_futex functions:
 - #dynamorio::drmemtrace::TRACE_MARKER_TYPE_FUNC_RETVAL
 - #dynamorio::drmemtrace::TRACE_MARKER_TYPE_FUNC_RETADDR
 
-Finally, every trace has a v2p.textproto file associated to it, which provides a
-plausible virtual to physical mapping of the virtual addresses present in a trace
-for more realistic TLB simulations.  This is a static virtual to physical mapping
-with 2 MB pages. Users can generate different mappings (e.g., smaller page size)
-by modifying this file, or create their own mapping following the same
-v2p.textproto format.
+Every trace has a v2p.textproto and an info.textproto file associated to it.
+
+The v2p.textproto file provides a plausible virtual to physical mapping of the virtual
+addresses present in a trace for more realistic TLB simulations.  This is a static
+virtual to physical mapping with 2 MB pages. Users can generate different mappings
+(e.g., smaller page size) by modifying this file, or create their own mapping following
+the same v2p.textproto format.
+
+The info.textproto file provides users with additional, human-readable information about
+a trace.  This file is not intended to be used with any DynamoRIO's tool.
 
 These traces are supported starting from DynamoRIO 11.3.
```

Comment:
Technically this is not true anymore, the user would need to move `info.textproto` out of the trace directory to have DR 11.3 work out of the box.

Maybe I should just remove the statement?

Should we spend some time on how to avoid this issue in the future?
It already happened for `v2p.textproto`.
IMHO it's not ideal having to release a DR minor version every time a new file is added to a trace directory.

Reply:
Agreed, switch to an allowlist of supported compression extensions?  Seems best to that now in this PR to avoid any more new releases?

Reply:
> Agreed, switch to an allowlist of supported compression extensions? 

Or can we move the aux files to their own separate directory inside the trace dir?

Reply:
I like Abhinav's suggestion.
Went with it in #7430.
I'm separating the two issues: 1) allowing additional files in the trace dir without having to bump DR's versions (#7430), 2) updating public v2 doc about `info.textproto` (this PR).